XenAdmin/TabPages/GeneralTabPage.cs

```diff
@@ -1724,59 +1768,6 @@ private static string HVMBootMode(VM vm)
 
         #endregion
 
-        #region Pool delegates
-
-        private string poolAppliedPatches()
-        {
-            return 
-                Helpers.ElyOrGreater(xenObject.Connection)
-                ? poolUpdateString(update => update.AppliedOnHosts().Count == xenObject.Connection.Cache.HostCount)
-                : poolPatchString(patch => patch.host_patches.Count == xenObject.Connection.Cache.HostCount);
-        }
-
-        private string poolPartialPatches()
-        {
-            return Helpers.ElyOrGreater(xenObject.Connection)
-                ? poolUpdateString(update =>
-                {
-                    var appliedOnHosts = update.AppliedOnHosts();
-                    return appliedOnHosts.Count > 0 && appliedOnHosts.Count != xenObject.Connection.Cache.HostCount;
-                })
-                : poolPatchString(patch => patch.host_patches.Count > 0 && patch.host_patches.Count != xenObject.Connection.Cache.HostCount);
-        }
-
-        private string poolPatchString(Predicate<Pool_patch> predicate)
-        {
-            Pool_patch[] patches = xenObject.Connection.Cache.Pool_patches;
-
-            List<String> output = new List<String>();
-
-            foreach (Pool_patch patch in patches)
-                if (predicate(patch))
-                    output.Add(patch.name_label);
-
-            output.Sort(StringUtility.NaturalCompare);

```

Comment:
nit: only detail is that it looks like we're losing this ordering. could it be an issue in terms of UX?

Reply:
There is ordering applied, see lines 621, 627, 634, is it not working?

Reply:
Looks like I missed it! 😅